.github/workflows/deploy-to-cloud-run.yml

```
@@ -0,0 +1,136 @@
+
+      - name: Push Docker image
+        run: |
+          docker push ${{ secrets.GCP_REGION }}-docker.pkg.dev/${{ secrets.GCP_PROJECT_ID }}/geo-tracking-repo/geo-tracking-service:${{ github.sha }}
+          
+      - name: Install Cloud SQL Proxy
+        run: |
+          # Download and install the Cloud SQL Auth Proxy
+          curl -o cloud-sql-proxy https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v2.16.0/cloud-sql-proxy.linux.amd64https://storage.googleapis.com/cloud-sql-connectors/cloud-sql-proxy/v2.8.1/cloud-sql-proxy.linux.amd64
+          chmod +x cloud_sql_proxy
+          sudo mv cloud_sql_proxy /usr/local/bin/
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)